ui.frontend/src/view/FormFieldBase.js

```diff
@@ -444,24 +441,6 @@ class FormFieldBase extends FormField {
         }
     }
 
-    /**
-     * Updates the HTML state based on the enabled state of the field.
-     * @param {boolean} enabled - The enabled state.
-     * @param {Object} state - The state object.
-     */
-    updateEnabled(enabled, state) {
```

Comment:
You had to keep this function.

you just had to remove the following code from this,

```
this.widget.setAttribute(Constants.ARIA_DISABLED, true);
this.widget.removeAttribute(Constants.ARIA_DISABLED);
```

Reply:
Done!